HomeworkLes5.java

```diff
@@ -0,0 +1,66 @@
+public class Worker {
+    private String name;
+    private String post;
+    private String email;
+    private String tel;
+    private int salary;
+    private int age;
+
+    Worker(String name, String post, String email, String tel, int salary, int age) {
+        this.name = name;
+        this.post = post;
+        this.email = email;
+        this.tel = tel;
+        this.salary = salary;
+        this.age = age;
+
+        if (age >= 18) {
+            this.age = age;
+        } else {
+            System.out.println( "работник несовершеннолетний!" );
+        }
+    }
+
+    int GetAge () {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.